src/codegen/compile.ml

```diff
@@ -5745,7 +5745,7 @@ module FuncDec = struct
         (* Look up closure *)
         let (set_closure, get_closure) = new_local env "closure" in
```

Comment:
Rename these too.

Reply:
No, turns out these are actually closures, already extracted from the continuation array.